File: Python Word examples/tables/get_table_row_format.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
inputFolder = "input"
table_path = "sections/0/tables/0"
index = 0
result = api.get_table_row_format(filename, table_path, index, folder=inputFolder)

File: JavaScript Word Examples/TablesApiDemo/Rows/getTableRowFormat.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient); 

    var filename = "Table.docx";
    var tablePath = "Section/0/Body/0/Table/0";
    var opt = {folder: 'Tables'};
    var index = 0;
    instance.getTableRowFormat(filename, tablePath, index, opt,
    function (error, data, response) {
         if (error) throw error;
    });
})();